src/client/theme-default/components/VPDocOutlineItem.vue

```diff
@@ -8,9 +8,7 @@ defineProps<{
 
 function onClick({ target: el }: Event) {
   const id = '#' + (el as HTMLAnchorElement).href!.split('#')[1]
-  const heading = document.querySelector<HTMLAnchorElement>(
-    decodeURIComponent(id)
-  )
+  const heading = document.getElementById(decodeURIComponent(id).slice(1))
```

Comment:
What about nod adding the `#` here and passing that to decodeURIComponent instead?

Reply:
ah yeah, fixed in 00d94e9